src/locales/pt-br.ts

```diff
@@ -51,48 +51,47 @@ export default {
     WHITE: 'branco',
```

Comment:
Some documents that could help to proof the translation:
- https://pt.wikipedia.org/wiki/Cores_lit%C3%BArgicas_na_Igreja_Cat%C3%B3lica_Apost%C3%B3lica_Romana
- http://www.liturgia.pt/edrel/pdf/IGMR_Sinopse.pdf

Reply:
In addition to the [IGMR Synopis (pt)](http://www.liturgia.pt/edrel/pdf/IGMR_Sinopse.pdf) that @emagnier shared with us, I have found full text of the [IGMR](https://www.arquidiocesedegoiania.org.br/download.php?arquivo=uploads/instrucao-geral-do-missal-romano-0562622.pdf&nome=instrucao-geral-do-missal-romano-0562622.pdf).

That said, @emagnier might be right that we should primarily use the text from GIRM (preferably from ed. typ. tertia), GILH and/or GILY (translated into the particular vernacular language) and not the Ordos (directories / liturgical calendars) issued by the particular episcopal conference and/or its liturgical commission.

Reply:
Currently in `pt-br`, all colours have `-o` suffix (it reminds me of neuter grammar gender) except for `rosa`. @HenriqueDerosa, should it be `roso` or `roséo`?

Reply:
Sorry for the delay guys!
Yes, it can be either `rosa` or `róseo`.
Not `roséo` @tukusejssirs .
Usually missal and other liturgic books has it described as **`roséo`**.